IonDotnet/Tree/IonSymbol.cs

```diff
@@ -5,6 +5,7 @@ namespace IonDotnet.Tree
     public sealed class IonSymbol : IonText
     {
         private int _sid;
+        private ImportLocation _importLocation;
```

Comment:
If ``_importLocation.Sid`` is the same as ``_sid`` and always present, can we eliminate ``_sid`` and use ``_importLocation.Sid`` instead?

Reply:
According to Avneet:

> Symbols will only have an import location if they were declared in a shared symbol table. And symbols with an import location will only have a local SID in the context of a local symbol table. Note that the developer guide does not actually require the local SID, although this is potentially a performant way to cache the value when it is known that the symbol will be written in the same local symbol table context.